.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+[pkg_name] update to 0.0.0
+
+[Release Notes](https://example.com)
+
+## Testing
+### Linux
+```
+hab pkg build pkg_name
+source results/last_build.env
+hab studio run "./pkg_name/tests/test.sh ${pkg_ident}"
+```
+### Windows
+```
+hab pkg build pkg_name
+source results/last_build.ps1
+hab studio run "./pkg_name/tests/test.ps1 $pkg_ident"
+```
+
+## Sample Output
+```
+ ✓ example test 1
+ ✓ example test 2
+
+2 tests, 0 failures
+```
+
+---------------------------------------------------
+Contributing Checklist
+- [ ] Read CONTRIBUTING.md
+- [ ] Run pre-commit
```

Comment:
It'd be helpful to link to what this means.
Note that https://github.com/habitat-sh/core-plans/pull/2767/files changes the pre-commit checks.